docs/LoaderInterfaceArchitecture.md

```diff
@@ -0,0 +1,694 @@
+        provided delimited folders to search for layer Manifest files.<br/>
+        <b>Ignored when running Vulkan application in executing with
+        elevated privileges.</b>
+        See <a href="#elevated-privilege-caveats">Elevated Privilege Caveats</a>
+        for more information.
+    </td>
+    <td><small>export<br/>
+        &nbsp;&nbsp;VK_LAYER_PATH=<br/>
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;path_a&gt;;&lt;path_b&gt;<br/><br/>
+        set<br/>
+        &nbsp;&nbsp;VK_LAYER_PATH=<br/>
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;path_a&gt;;&lt;path_b&gt;</small>
+    </td>
+  </tr>
+  <tr>
+    <td><small><i>VK_LOADER_DISABLE_INST_EXT_FILTER</i></small></td>
+    <td>Disable the filtering out of instance extensions that the loader doesn't
+        know about.
+        This will allow applications to enable instance extensions exposed by
+        implementations but that the loader has no support for.<br/>
+        <b>NOTE:</b> This may cause the loader or application to crash.</td>
+    <td><small>export<br/>
+        &nbsp;&nbsp;VK_LOADER_DISABLE_INST_EXT_FILTER=1<br/><br/>
+        set<br/>
+        &nbsp;&nbsp;VK_LOADER_DISABLE_INST_EXT_FILTER=1</small>
+    </td>
+  </tr>
+  <tr>
+    <td><small><i>VK_LOADER_DEBUG</i></small></td>
+    <td>Enable loader debug messages using a comma-delimited list of level
+        options.  These options are:<br/>
+        &nbsp;&nbsp;* error (only errors)<br/>
+        &nbsp;&nbsp;* warn (only warnings)<br/>
+        &nbsp;&nbsp;* info (only info)<br/>
+        &nbsp;&nbsp;* debug (only debug)<br/>
+        &nbsp;&nbsp;* all (report out all messages)<br/><br/>
+        To enable multiple options (outside of "all") like info, warning and
+        error messages, set the value to "error,warn,info".
+    </td>
+    <td><small>export<br/>
+        &nbsp;&nbsp;VK_LOADER_DEBUG=all<br/>
+        <br/>
+        set<br/>
+        &nbsp;&nbsp;VK_LOADER_DEBUG=warn</small>
+    </td>
+  </tr>
+</table>
+<br/>
+<br/>
+
+## Glossary of Terms
+
+<table style="width:100%">
+  <tr>
+    <th>Field Name</th>
+    <th>Field Value</th>
+  </tr>
+  <tr>
+    <td>Android Loader</td>
+    <td>The loader designed to work primarily for the Android OS.
+        This is generated from a different code base than the desktop loader.
+        But, in all important aspects, it should be functionally equivalent.
+    </td>
+  </tr>
+  <tr>
+    <td>Desktop Loader</td>
+    <td>The loader designed to work on Windows, Linux and macOS.
+        This is generated from a different
+        <a href="https://github.com/KhronosGroup/Vulkan-Loader">code base</a>
+        than the Android loader.
+        But in all important aspects, it should be functionally equivalent.
+    </td>
+  </tr>
+  <tr>
+    <td>Core Function</td>
+    <td>A function that is already part of the Vulkan core specification and not
+        an extension. <br/>
+        For example, <b>vkCreateDevice()</b>.
```

Comment:
Is it worth adding the description "Extension functions always have a suffix to denote which type of extension they are, such as KHR and EXT"?